contracts/multisig-prover/src/contract/execute.rs

```diff
@@ -34,6 +40,14 @@ pub fn construct_proof(
         config.chain_name.clone(),
     )?;
 
+    // Error in case we have messages from ITS Hub
+    if let Some(_) = messages.iter().find(|msg| {
+        msg.cc_id.source_chain.as_ref() == AXELAR_CHAIN_NAME
+            && msg.source_address.as_str() == config.its_hub_address.as_str()
+    }) {
+        return Err(ContractError::InvalidMessages.into());
+    }
```

Comment:
same comment here as in the comment about verify_messages. I think we should just remove this whole function

Reply:
Done